go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/google/go-cmp v0.4.0 // indirect
 	github.com/imdario/mergo v0.3.8 // indirect
 	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
+	github.com/mattn/go-isatty v0.0.8
```

Comment:
I don't _love_ this dependency in the API repo. Would anyone complain too hard if we didn't colorize the output, ever?

Reply:
I'm ok with either removing tty checks completely, or changing the marshal function signature to `MarshalText(...MarshalOption)` where `WithColor` is an option (no color by default). Callers can import the tty package.

Reply:
Regardless we can deal with this in a follow-up.